### DISCUSSION TITLE
Random "Crashes?"

### DISCUSSION THREAD

**Discussion by Stoppedpuma** [ppy/osu#26210] [Q&A]
### Type

Crash to desktop

### Bug description

I have no idea where to start with this but it's an issue I've been facing for months, osu! will randomly just minimize and continue playing audio and never come back. I call it a crash because the closest things I can get to a log containing information about the crash is something like this `tmp-.mount_osu.ApUuFaCZ.mount: Deactivated successfully.` being reported in journalctl, and sometimes when I run the game in my terminal with the following start command `gamemoderun SDL_VIDEODRIVER=WAYLAND ~/bin/osu.Appimage` I get the following output during the crash but it's like one out of every ten or so times. (Variables like gamemoderun and sdl don't cause the crash as I get it without them as well.)

`ERROR: ld.so: object 'libgamemodeauto.so.0' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
ERROR: ld.so: object 'libgamemodeauto.so.0' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
dbus[1603329]: arguments to dbus_pending_call_block() were incorrect, assertion "pending != NULL" failed in file dbus-pending-call.c line 766.
This is normally a bug in some application using the D-Bus library. D-Bus not built with -rdynamic so unable to print a backtrace`

This issue mostly occurs during higher than usual cpu usage, the lowest I remember it happening with is around 38% but I'm sure it's happened lower and I just never bothered to look. I'm opening here instead of discussions as I don't believe it's a hardware / software configuration issue as I've been able to this replicate on multiple machines running different linux based operating systems as well as seen it happen to friends.

### Screenshots or videos

[lazer_crash_issue.webm](https://github.com/ppy/osu/assets/58333920/7a502389-8a10-4216-ae70-06e66d491e95)

### Version

2023.1229.0

### Logs

[shorter_runtime.log](https://github.com/ppy/osu/files/13790769/shorter_runtime.log)
[shorter_performance.log](https://github.com/ppy/osu/files/13790770/shorter_performance.log)
[shorter_network.log](https://github.com/ppy/osu/files/13790771/shorter_network.log)
[shorter_database.log](https://github.com/ppy/osu/files/13790772/shorter_database.log)

**Comment by Stoppedpuma** [comment]
I just got a coredump after a slightly different crash happened around an hour ago:

Message: Process 3064425 (osu!) of user 1000 dumped core.
                
                Module /tmp/.mount_osu.ApDAJ5XC/usr/bin/librealm-wrappers.so without build-id.
                Module /tmp/.mount_osu.ApDAJ5XC/usr/bin/librealm-wrappers.so
                Module /tmp/.mount_osu.ApDAJ5XC/usr/bin/libbassmix.so without build-id.
                Module /tmp/.mount_osu.ApDAJ5XC/usr/bin/libbassmix.so
                Module /tmp/.mount_osu.ApDAJ5XC/usr/bin/libbass_fx.so without build-id.
                Module /tmp/.mount_osu.ApDAJ5XC/usr/bin/libbass_fx.so
                Module /tmp/.mount_osu.ApDAJ5XC/usr/bin/libbass.so without build-id.
                Module /tmp/.mount_osu.ApDAJ5XC/usr/bin/libbass.so
                Stack trace of thread 3066940:
                #0  0x00007f1dd2e2a0ca n/a (/tmp/.mount_osu.ApDAJ5XC/usr/bin/libbass.so + 0x2a0ca)
                ELF object binary architecture: AMD x86-64
                
 (gdb) backtrace
#0  0x00007f1dd2e2a0ca in ?? ()
#1  0x00007f1eac9ba700 in ?? ()
#2  0x0000000000000000 in ?? ()

**Comment by peppy** [comment]
I'm guessing (not helped by the fact you're providing incomplete log files here) that you're a linux user. This will be on you to figure out. Likely something on your configuration since no one else reports this.
